aait/web/group-4/starter-project/components/story/ShowStories.tsx

```diff
@@ -0,0 +1,38 @@
+import { Story } from "@/types/Story/StoryType";
+import { useGetAllStoriesQuery } from "@/store/story/StoryApi";
+import StoryCard from './StoryCard'
+import Partners from "./Partners";
+
+export default function showStories() {
+    const{data:allStories, isLoading, isError} = useGetAllStoriesQuery()
+     
+    if(isError)
+    {
+        return <h1>Error: {isError.toString()}</h1>
+    }
+
+    if(allStories === undefined || isLoading){
```

Comment:
`isLoading` is enough in here. This is because the API call would have completed when `allStories` is populated, even if it's an empty array.